grumphp.yml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  tasks:
+    git_commit_message:
+      allow_empty_message: false
+      enforce_capitalized_subject: false
+      enforce_no_subject_trailing_period: true
+      enforce_no_subject_punctuations: true
```

Comment:
This one can be frustrating. There are times when a comma, semicolon, or other non-period/full-stop character is appropriate and this rule prevents it.

Reply:
To add to this, I'm pretty sure that we discussed this in the past and agreed that it would simply be better to prevent punctuation only at the *end* of the line (this was in the `web-api` repo, though).

Reply:
I just did the same rules we had for web-api